op-challenger/game/fault/register.go

```diff
@@ -163,14 +166,18 @@ func (r *outputCannonResources) Contract() GameContract {
 	return r.contract
 }
 
+func (r *outputCannonResources) validateAbsolutePrestate(ctx context.Context, provider faultTypes.TraceProvider) error {
+	// TODO(client-pod#260): Use the OutputBisectionGame Contract and then use the genesis output root here instead of the absolute prestate hash method
+	return ValidateAbsolutePrestate(ctx, provider, r.contract.GetAbsolutePrestateHash)
```

Comment:
Can we close out 260 here w/ the new `GetGenesisOutputRoot` func?

Reply:
Yes, once this pr is rebased ontop of develop once https://github.com/ethereum-optimism/optimism/pull/8379 is merged (dang merge queue *old man shakes fist*)